tests/docker/install_local_irods_client_packages_ubuntu20.sh

```diff
@@ -0,0 +1,2 @@
+# To test unreleased versions of iRODS, build the iRODS packages, uncomment the following line, and update the version as necessary.
+#apt-get install -y /irods_packages/irods-dev_4.3.3-0~focal_amd64.deb /irods_packages/irods-runtime_4.3.3-0~focal_amd64.deb /irods_packages/irods-icommands_4.3.3-0~focal_amd64.deb
```

Comment:
You probably want to run `apt-get update` before `apt-get install` here

Reply:
done